### PR TITLE
chore: fix types to be compatible with typescript 3.1 and node 10.x

### PR DESCRIPTION
**Pull request by raymondfeng**
CI is failing due to TypeScript 3.1 and latest Node.js types

## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: packages/repository/src/repositories/kv.repository.bridge.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ import {Entity} from '../model';
 import {KeyValueRepository, KeyValueFilter} from './kv.repository';
 
 import {juggler, ensurePromise} from './legacy-juggler-bridge';
+import {DeepPartial} from 'loopback-datasource-juggler';
```
Comment (virkt25) [inline packages/repository/src/repositories/kv.repository.bridge.ts]: This isn't used anywhere in the file. Is it needed?

Comment (raymondfeng) [reply packages/repository/src/repositories/kv.repository.bridge.ts]: It was by mistake.

**Comment by raymondfeng** [comment]
One more issue to figure out:
```
examples/rpc-server/node_modules/@types/node/index.d.ts(198,15): error TS2451: Cannot redeclare block-scoped variable 'console'.
packages/build/node_modules/typescript/lib/lib.dom.d.ts(17295,13): error TS2451: Cannot redeclare block-scoped variable 'console'.
```

**Comment by raymondfeng** [comment]
See https://github.com/DefinitelyTyped/DefinitelyTyped/issues/29228

**Comment by raymondfeng** [comment]
@virkt25 I updated it to be `"@types/node": "^10.11.2",` so that there is no chance to install the broken version.

**Pull request merged**